docs/examples/drag-and-drop/README.md

```diff
@@ -0,0 +1,258 @@
+    background: var(--ag-theme-background);
+    color: var(--ag-theme-text);
+    font-family: var(--ag-theme-font-family);
+}
+#uploadBox {
+    padding: 1em;
+    border: 0.25em solid var(--ag-theme-primary);
+    border-radius: 0.5em;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+a {
+    color: var(--ag-theme-secondary);
+    padding:0.15em 0.5em;
+    border-radius: 0.5em;
+}
+a:hover {
+    color: var(--ag-theme-background);
+    background-color: var(--ag-theme-secondary);
+    text-decoration: none;
+}		
+main {
+    margin: 1em;
+}
+select, select * {
+    background: var(--ag-theme-background);
+    font-size: inherit;
+    margin-bottom: 1em;
+    color: var(--ag-theme-text);
+    border: 0.05em solid var(--ag-theme-secondary);
+    padding:0.25em;
+    border-radius: 0.25em;
+}
+*::selection, option:hover {
+    background: var(--ag-theme-primary);
+    color: var(--ag-theme-text);
+}
+```
+- In our `styles.css`, you'll notice the line `@import url("agregore://theme/vars.css");`. This imports a set of CSS variables defined by the Agregore browser, allowing us to use consistent theming across our application. These variables include colors, font family, and more, which are used to style various elements of our app.
+
+- By using these variables, our app's look and feel will align with the user's configured theme in the Agregore browser, providing a seamless and integrated experience.
+
+- For more detailed information on theming in Agregore, check out the [Agregore Theming Documentation](https://agregore.mauve.moe/docs/theming).
+
+### Step 3: JavaScript for Drag-and-Drop
+In your `script.js` file, add the JavaScript code:  
+
+```javascript
+function $(query) {
+    return document.querySelector(query)
+}
+
+const uploadBox = $('#uploadBox')
+uploadBox.ondragover = () => false
+uploadBox.ondrop = async (e) => {
+    e.preventDefault()
+    const { dataTransfer } = e
+    if(!dataTransfer) return
+
+    for(const file of dataTransfer.files) {
+        await uploadFile(file)
+    }
+}
+
+const uploadListBox = $('#uploadListBox')
+const protocolSelect = $('#protocolSelect')
+
+async function uploadFile(file) {
+    const name = file.name;
+    const buffer = await file.arrayBuffer();
+    const protocol = protocolSelect.value;
+
+    // Create a Blob from the ArrayBuffer
+    let mimeType = 'application/octet-stream'; // Default MIME type
+    // Use the MIME type from the File object if available
+    if (file instanceof File) {
+        mimeType = file.type || mimeType;
+    }
+
+    const blob = new Blob([buffer], { type: mimeType }); // Create a blob from the array buffer
+
+    // Headers
+    const headers = {
+        'Content-Type': mimeType,
+    };
+
+    // Construct the URL with the actual hypercore key
+    let url;
+    let hyperdriveUrl;
+    let body = file instanceof File ? blob : buffer;
+
+    if (protocol === 'hyper') {
+        hyperdriveUrl = await generateHyperdriveKey(name);
```

Comment:
Usually we use one drive per "app". Might be more space efficient to use a single "drag-n-drop" drive and put all your files into it.

Reply:
Now using a signe 'drag-and-drop' drive with `generateHyperdriveKey('drag-and-drop')`